lib/stemcell/option_parser.rb

```diff
@@ -261,6 +261,12 @@ class OptionParser
         :env   => 'NON_INTERACTIVE',
         :short => :f
       }
+      {
+        :name  => 'context',
+        :desc  => "comma delimited string for specifying contexts to override certain values",
```

Comment:
Should this be `contexts` (plural) for consistency with `ephemeral_devices`? Also maybe change the description to say `comma-separated list` instead of `comma delimited string`, again for consistency.